src/index.d.ts

```diff
@@ -0,0 +1,143 @@
+/**
+ * Type definitions for @thundra/core v2.12.4
+ *
+ * Project:     Thundra Lambda Node.js Agent
+ * Author:      Oguzhan Ozdemir <oguzhan.ozdemir@thundra.io>
+ */
+
+declare module '@thundra/core' {
+  /**
+   * Initialized agent with given options (configs, etc ...)
+   * @param options the options (configs, etc ...) to initialize agent
+   */
+  export function init(options?: any): void;
+
+  /**
+   * Creates {@link LambdaWrapper} to wrap the original AWS Lambda handler
+   * @param options the options (configs, etc ...) to initialize agent
+   * @return {LambdaWrapper} the AWS Lambda wrapper to wrap the original handler
+   */
+  export function createLambdaWrapper(options?: any): (f: Function) => WrappedFunction;
+
+  /**
+   * Wraps the given original AWS Lambda handler
+   * @param handler the original AWS Lambda handler to be wrapped
+   * @return the wrapped handler
+   */
+  export function lambdaWrapper(handler: any): (f: Function) => WrappedFunction;
+
+  /**
+   * Creates {@link Logger} with given options
+   * @param options the options (configs, etc ...) to initialize logger to be created
+   */
+  export function createLogger(options: any): Logger;
+
+  /**
+   * Creates an Express.js middleware to integrate Thundra
+   * @return the Thundra middleware
+   */
+  export function expressMW(): void;
+
+  /**
+   * Adds given log listener
+   * @param listener the log listener to be added
+   */
+  export function addLogListener(listener: any): void;
+
+  /**
+   * Gets the tracer
+   * @return {Tracer} the tracer
+   */
+  export function tracer(): ThundraTracer;
+
+  /**
+   * Loads and returns the user AWS Lambda handler
+   * @return the loaded user AWS Lambda handler
+   */
+  export function loadUserHandler(): any;
```

Comment:
I'm not sure about this one. `loadUserHandler` calls the `loadHandler`. However, `loadHandler` doesn't have any return value in the code, which confused me a little. Giving `any` as a return value something I didn't want to do. Open to suggestions.